src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/FrameOfT.cs

```diff
@@ -31,66 +32,84 @@ public class Frame<TContext> : Frame
         /// </summary>
         public override async Task RequestProcessingAsync()
         {
-            var requestLineStatus = RequestLineStatus.Empty;
+            var requestLineStatus = default(RequestLineStatus);
 
             try
             {
                 while (!_requestProcessingStopping)
                 {
+                    requestLineStatus = RequestLineStatus.Empty;
```

Comment:
Maybe a comment on why this need to be reset in the loop before the call to TakeStartLine.